docs/apps/lnw/es2k/es2k-lnw-overlay-ipv6.md

```diff
@@ -100,12 +103,14 @@ These VSI values can be checked with `/usr/bin/cli_client -q -c` command on IMC.
 Once the application is started, set the forwarding pipeline config using
 P4Runtime Client `p4rt-ctl` set-pipe command
 
-```bash
-$P4CP_INSTALL/bin/p4rt-ctl set-pipe br0 $OUTPUT_DIR/fxp-net_linux-networking-v2.pb.bin \
-    $OUTPUT_DIR/p4info.txt
-```
+- For Linux Networking v3
+
+  ```bash
```

Comment:
Code block should be aligned with the opening fence.

  \`\`\`bash
  ^ line starts here